assets/agenda/components/AgendaDropdown.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {gettext} from 'utils';
+import {Dropdown} from '../../components/Dropdown';
+
+export function AgendaDropdown({filter, activeFilter, toggleFilter, children}) {
+
+    const isActive = activeFilter[filter.field];
+
+    const getActiveFilterLabel = (filter, activeFilter, isActive) => {
```

Comment:
There probably isn't a need for this to be a function, as we're calling it straight away and using the value.
This could be replaced with a variable assignment instead.